File: dotnet/niuz.application/hr/HireAuthor.cs
namespace niuz.application.hr
{
    public class HireAuthor
    {
        public string AuthorId { get; }
        public string AuthorName { get; }
        public string ContractType { get; }
        public int Rate { get; }
        public string BankAccount { get; }

        public HireAuthor(string authorId, string authorName, string contractType, int rate, string bankAccount)
        {
            AuthorId = authorId;
            AuthorName = authorName;
            ContractType = contractType;
            Rate = rate;
            BankAccount = bankAccount;
        }
    }
}

File: java/src/main/java/com/dddeurope/niuz/hr/HireAuthor.java
package com.dddeurope.niuz.hr;

public class HireAuthor {
    private final String authorId;
    private final String authorName;
    private final String contractType;
    private final int rate;
    private final String bankAccount;

    public HireAuthor(String authorId, String authorName, String contractType, int rate, String bankAccount) {
        this.authorId = authorId;
        this.authorName = authorName;
        this.contractType = contractType;
        this.rate = rate;
        this.bankAccount = bankAccount;
    }

    public String getAuthorId() {
        return authorId;
    }

    public String getAuthorName() {
        return authorName;
    }

    public String getContractType() {
        return contractType;
    }

    public int getRate() {
        return rate;
    }

    public String getBankAccount() {
        return bankAccount;
    }
}
